openapi.yml

```diff
@@ -5569,6 +5596,143 @@ components:
                 pagination:
                     $ref: '#/components/schemas/Pagination'
                     description: ''
+        Registration:
+            title: Root Type for Registration
+            description: ''
+            type: object
+            properties:
+                id:
+                    format: int32
+                    description: Unique identifier of `Registration`
+                    type: integer
+                created_at:
+                    description: Timestamp when this `Registration` was created
+                account_id:
+                    description: Unique identifier of the `Account` this `Registration` belongs to
+                    type: integer
+                visitor_id:
+                    description: Unique identifier of the `Visitor` this `Registration` belongs to
+                    type: integer
+                pre_registration_id:
```

Comment:
The term `pre_registration` doesn't exist anywhere in APIv3. We're using `invite` here.

Reply:
Ya, any reference to a PreRegistration needs to be removed